sql/updates/world/3.3.5/9999_99_99_99_world_335.sql

```diff
@@ -0,0 +1,4 @@
+DELETE FROM `spell_script_names` WHERE `spell_id`= 52267;
+DELETE FROM `spell_script_names` WHERE `ScriptName`='spell_deliver_stolen_horse';
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
+(52264, 'spell_deliver_stolen_horse');
```

Comment:
missing new line.

Reply:
Yes, just add 1 `<Enter>` at the end of the file to follow TC codestyle.